numpy/core/src/umath/loops_arithmetic.dispatch.c.src

```diff
@@ -0,0 +1,132 @@
+/*@targets
+ ** $maxopt baseline
+ ** sse2 sse41 avx2 avx512_skx
+ ** vsx2
+ ** neon
+ **/
+#define _UMATHMODULE
+#define _MULTIARRAYMODULE
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+
+#include "simd/simd.h"
+#include "loops_utils.h"
+#include "loops.h"
+#include "lowlevel_strided_loops.h"
+#include<signal.h> 
+// Provides the various *_LOOP macros
+#include "fast_loop_macros.h"
+
+//###############################################################################
+//## Unsigned Integers
+//###############################################################################
+/********************************************************************************
+ ** Defining the SIMD kernels
+ ********************************************************************************/
+#ifdef NPY_SIMD
+/**begin repeat
+ *  #sfx = u8, u16, u32, u64#
+ *  #type = npy_ubyte, npy_ushort, npy_uint, npy_ulong#
+ */
+
+static void simd_divide_by_scalar_contig_contig_@sfx@
+(const char *src, const @type@ scalar, char *dst,
+ int len)
+{
+    npyv_lanetype_@sfx@ *npyv_src = (npyv_lanetype_@sfx@ *) src;
+    npyv_lanetype_@sfx@ npyv_scalar = (npyv_lanetype_@sfx@) scalar;
+    npyv_lanetype_@sfx@ *npyv_dst = (npyv_lanetype_@sfx@ *) dst;
+    const int vstep = npyv_nlanes_@sfx@;
+    const npyv_@sfx@x3 divisor = npyv_divisor_@sfx@(npyv_scalar);
+
+    for (; len >= vstep; len -= vstep, npyv_src += vstep, npyv_dst += vstep) {
+        npyv_@sfx@ a = npyv_load_@sfx@(npyv_src);
+        npyv_@sfx@ c = npyv_divc_@sfx@(a, divisor);
+        npyv_store_@sfx@(npyv_dst, c);
+    }
+    for (; len > 0; --len, ++npyv_src, ++npyv_dst) {
+        const npyv_lanetype_@sfx@ a = *npyv_src;
+        *npyv_dst = a / npyv_scalar;
+    }
+    npyv_cleanup();
+}
+
+/**end repeat**/
+#endif
+
+/**begin repeat
+ * Unsigned types
+ *  #type = npy_ubyte, npy_ushort, npy_uint, npy_ulong, npy_ulonglong#
```

Comment:
```suggestion
 *  #type = npy_ubyte, npy_ushort, npy_uint, npy_ulonglong#
```
you can map the loop of `npy_ulong` to the loop of `npy_uint` or `npy_ulonglong` according to data size.